src/sql/azdata.proposed.d.ts

```diff
@@ -1061,6 +1061,14 @@ declare module 'azdata' {
 		 */
 		export type DesignerEditPath = (string | number)[];
 
+		/**
+		 * Severity of the messages returned by the provider after processing an edit.
+		 * 'error': The issue must be fixed in order to commit the changes.
+		 * 'warning': Inform the user the potential risks with the current state. e.g. Having multiple edge constraints is only useful as a temporary state.
+		 * 'information': Informational message.
```

Comment:
'informational' instead of 'information'?

Reply:
I'd say 'information' for the string is more common anyways, but whatever one you prefer